modelopt/onnx/trt_utils.py

```diff
@@ -334,6 +334,11 @@ def load_onnx_model(
         if intermediate_generated_files is not None:
             intermediate_generated_files.append(ir_version_onnx_path)
 
+    # Ensure nodes are topologically sorted
```

Comment:
What operation(s) make the graph nodes "un-sorted"?

Reply:
This sometimes happens during the model export to ONNX. See model 4 in table in bug 5591945@18.